src/assets/scss/components/_button-spinner.scss

```diff
@@ -1,21 +1,45 @@
+// spinner button styling
+
+$spinner-icon-size: 24px;
+$spinner-icon-path: "/public/images/mini-button-spinner.svg";
+$spinner-animation-duration: 0.75s;
+
+// Create variables that are in local scope for the design system version of the button
+// https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/button/_index.scss
+// IN GLOBAL SCOPE and reusable
+// $govuk-button-text-colour
```

Comment:
Might be missing something but how does this work if it's commented out?

Reply:
They don't work, they're just part of the notes that the person who wrote the file left. Shall I remove all the comments in the file or just those ones?